app/notifications/fork_notification.rb

```diff
@@ -4,9 +4,9 @@ class ForkNotification < Noticed::Base
   deliver_by :database, association: :noticed_notifications
 
   def message
-    user = params[:user]
-    project = params[:project]
-    t("users.notifications.fork_notification", user: user&.name, project: project&.name)
+    user = params[:user] || DeletedUser.new
+    project = params[:project] || DeletedUser.new
+    t("users.notifications.fork_notification", user: user.name, project: project.name)
```

Comment:
Could you apply the same logic for other notifications too ?

Reply:
Sure I do